openedx/core/djangoapps/catalog/utils.py

```diff
@@ -151,16 +173,108 @@ def get_program_types():
         return []
 
 
-def get_programs_with_type_logo():
+def _get_program_instructors(program):
```

Comment:
The instructor info we are pulling in here is not stored in the catalog service. It is stored in the modulestore. It is added via a UI in Studio. We considered updating the refresh_course_metadata command to pull this data into the catalog service, but decided it would be better to put this workaround into place for now until we can move course data entry to the publisher tool. Once we do that, we can start consuming instructor data directly from the catalog service.